<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UMRR3LS" name="NUGUAPP_003_홈.홈_01_홈카드" tests="12" failures="0" timestamp="16 Jun 2020 03:01:37 GMT" time="85.176" errors="1">
  <testcase name="TC_홈_001_07" time="6.829" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_12" time="10.063" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_10" time="1.505" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_06" time="6.232" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_11" time="7.586" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_05" time="6.963" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_01" time="18.233" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_03" time="5.700" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_02" time="5.577" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_08" time="6.817" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
  <testcase name="TC_홈_001_09" time="6.996" classname="NUGUAPP_003_홈.홈_01_홈카드">
    <error type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.assertTrue(Assert.java:20)
	at junit.framework.Assert.assertTrue(Assert.java:27)
	at NUGUAPP_003_홈.홈_01_홈카드.TC_홈_001_09(홈_01_홈카드.java:139)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- TC_홈_001_09 -->
  <testcase name="TC_홈_001_04" time="2.675" classname="NUGUAPP_003_홈.홈_01_홈카드"/>
</testsuite> <!-- NUGUAPP_003_홈.홈_01_홈카드 -->
